packages/tables/src/Filters/SelectFilter.php

```diff
@@ -18,7 +18,7 @@ public function apply(Builder $query, array $data = []): Builder
             return parent::apply($query, $data);
         }
 
-        if (! $data['value']) {
+        if ($data['value'] === null || $data['value'] === '') {
```

Comment:
Could we not use the `blank()` helper function here?

Reply:
Yup!